tests/Service/BroadcastsService/AbstractBroadcastsServiceTest.php

```diff
@@ -2,31 +2,31 @@
 
 namespace Tests\BBC\ProgrammesPagesService\Service\BroadcastsService;
 
+use BBC\ProgrammesPagesService\Domain\Entity\Broadcast;
+use BBC\ProgrammesPagesService\Domain\ValueObject\Pid;
 use BBC\ProgrammesPagesService\Service\BroadcastsService;
 use Tests\BBC\ProgrammesPagesService\AbstractServiceTest;
 
 abstract class AbstractBroadcastsServiceTest extends AbstractServiceTest
 {
-    public function setUp()
+    public function setUp(): void
     {
         $this->setUpCache();
         $this->setUpRepo('BroadcastRepository');
         $this->setUpMapper('BroadcastMapper', 'broadcastFromDbData');
     }
 
-    protected function broadcastsFromDbData(array $entities)
+    /**
+     * @param array[] $dbData
```

Comment:
If it's an array of arrays, how can you access `$dbData['pid']`? Is the value of `$dbData['pid']` an array?